Formula/uutils-findutils.rb

```diff
@@ -0,0 +1,38 @@
+class UutilsFindutils < Formula
+  desc "Cross-platform Rust rewrite of the GNU findutils"
+  homepage "https://github.com/uutils/findutils"
+  url "https://github.com/uutils/findutils/archive/refs/tags/0.3.0.tar.gz"
+  sha256 "0ea77daf31b9740cfecb06a9dbd06fcd50bc0ba55592a12b9f9b74f3302f5c41"
+  license "MIT"
+  head "https://github.com/uutils/findutils.git", branch: "main"
+
+  depends_on "rust" => :build
+  uses_from_macos "llvm"
```

Comment:
No linkage, so we can probably make this `:build`.

Reply:
```suggestion
  uses_from_macos "llvm" => :build
```